src/app/core/entity/entity-type-selector/entity-type-selector.component.ts

```diff
@@ -0,0 +1,32 @@
+import { Component, OnInit, Input } from "@angular/core";
+import { BasicAutocompleteComponent } from "../../common-components/basic-autocomplete/basic-autocomplete.component";
+import { MatFormField } from "@angular/material/form-field";
+import { ReactiveFormsModule } from "@angular/forms";
+import { EntityConstructor } from "../model/entity";
+import { EntityRegistry } from "../database-entity.decorator";
+
+/**
+ * Component for selecting an entity type from a dropdown.
+ */
+@Component({
+  selector: "app-entity-type-selector",
+  templateUrl: "./entity-type-selector.component.html",
+  imports: [BasicAutocompleteComponent, MatFormField, ReactiveFormsModule],
+  standalone: true,
+})
+export class EntityTypeSelectorComponent implements OnInit {
```

Comment:
I think we are now duplicating the same code that is also in EditEntityTypeDropdownComponent?

Can you change the EditEntityType... component to use this new component now, instead of keeping duplicate code?

Reply:
Yes, I have made a few changes to the `EditEntityTypeDropdownComponent`. Please take a look and let me know if there are any other changes needed.